backend/src/test/java/com/redhat/cloud/notifications/events/LifecycleITest.java

```diff
@@ -103,6 +104,8 @@ void test() {
         RequestEmailSubscriptionProperties userEmailEndpointRequest = new RequestEmailSubscriptionProperties();
 
         // Identity header used for all public APIs calls. Internal APIs calls need a different token.
+        // TODO should I add the orgId here as well?
```

Comment:
I guess so, in the future we are going to receive the orgId as a header as well

Reply:
woah. I just checked that change. It actually looks like we are using the accountId/account number everywhere :X

Reply:
It's used everywhere, yes.